ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-master__serverless-rehearse-4.12.yaml

```diff
@@ -33,7 +36,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: installer-rehearse-aws
+- as: serverless-rehearse-aws
   interval: 960h
   steps:
     cluster_profile: aws-cspi-qe
```

Comment:
Please use the env var `SPOT_INSTANCES: "true"` when testing so that we save AWS costs.

Reply:
Yes! Thank you @ascerra for pointing this out when doing testing when it does not require the standard AWS instance types.

https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml#L40-L42